app/views/ineligible_patient_mailer/no_managed_care_patients.haml

```diff
@@ -0,0 +1,7 @@
+%h2 Patient MassHealth Eligibility
+
+In a recent MassHealth eligibility check, the following patients were marked as not having managed care coverage:
+
+%ul
+  - @patients.each do |patient|
+    %li= link_to nil, window_client_health_patient_index_url(patient)
```

Comment:
There are some helper methods for these and polymorphic path.  We should use those.  See app/controllers/concerns/combined_client_paths_generator.rb